Annotato/Annotato/Persistence/PersistenceService/LocalPersistenceService/DataAccess/LocalDocumentEntityDataAccess.swift

```diff
@@ -157,14 +157,14 @@ struct LocalDocumentEntityDataAccess {
         }
     }
 
-    func delete(documentId: UUID, document: Document) -> DocumentEntity? {
+    func delete(documentId: UUID, document: Document) {
```

Comment:
Why is this behavior changed? Is it because it is hard deleted? I would assume that it still returns me the hard deleted document entity.